src/qt/receivecoinsdialog.cpp

```diff
@@ -72,6 +72,9 @@ void ReceiveCoinsDialog::setModel(WalletModel *_model)
 {
     this->model = _model;
 
+        setWindowTitle(tr("Request payments - %1").arg(model->getWalletName()));
```

Comment:
CI lint is [failing](https://github.com/bitcoin-core/gui/pull/786/checks?check_run_id=20356215277) due to unnecessary white-spaces in the files (you can run the script `lint-whitespace.py` locally or even [all linters](https://github.com/bitcoin/bitcoin/blob/master/test/lint/README.md#running-locally))
```suggestion
    setWindowTitle(tr("Request payments - %1").arg(model->getWalletName()));
```